go/oasis-node/cmd/debug/txsource/workload/queries.go

```diff
@@ -69,6 +72,87 @@ type queries struct {
 	runtimeGenesisRound uint64
 }
 
+func (q *queries) sanityCheckTransactionEvents(ctx context.Context, height int64, txEvents []*results.Event) error {
+	// Ensure transaction events match querying backend GetEvents methods.
+	// TODO: Registry events don't include any transaction info, so we cannot
```

Comment:
Should we add this (e.g. adding some tx info - maybe the transaction hash as we do for staking, to registry events as well)?

Reply:
Yeah we should.